doc/source/user_guide/fields/field_data.rst

```diff
@@ -85,7 +85,7 @@ To obtain face connectivity data, specify ``FacesConnectivity`` as the ``data_ty
   >>> surface_request_3 = SurfaceFieldDataRequest(
   >>>     surfaces=["inlet"], data_types=[SurfaceDataType.FacesConnectivity]
   >>> )
-  >>> faces_connectivity_data = field_data.get_field_data(surface_request_3)
+  >>> faces_connectivity_data = field_data.get_field(surface_request_3)
```

Comment:
@prmukherj `get_field()` returns a nested dictionary.

Reply:
Shall we revert to the original name or is there any other suggestion?
Some suggestions:
'get_data', 'retrieve_field_data'

Reply:
I think we can revert to the 'get_field_data' only.

Reply:
Done.